plaid-stl/src/messages.rs

```diff
@@ -5,6 +5,7 @@ pub enum Generator {
     Github,
     Okta,
     Interval(String),
+    Websocket(String),
```

Comment:
Probably `WebSocket`

Reply:
We also might want to call this `WebSocketExternal` because in the future we may want to add the ability to host websockets for people to send logs to.